src/prefect/results.py

```diff
@@ -49,6 +50,13 @@ def get_default_result_serializer() -> ResultSerializer:
     return PREFECT_RESULTS_DEFAULT_SERIALIZER.value()
 
 
+def get_default_persist_setting() -> bool:
```

Comment:
I don't have a suggestion for a better name, but this makes me think that we're getting the default setting, a setting a user could override and not the 'default' set by the user. Any reason to not just use `PREFECT_RESULTS_PERSIST_BY_DEFAULT.value()` directly?

Reply:
I could use it directly, but I'm following this pattern because we use it for all the other default values. We are likely to extend these to pull settings from the API in the future. I can try to think of a clearer name.

Reply:
I don't really have a good idea and since all three of these `get_default_...` methods read settings I'm going to leave it as is for now.